### PR TITLE
[stable/gocd] - GoCD agents should run postStart lifecycle hooks

### PR DESCRIPTION
**Pull request by igno**
Signed-off-by: Erik Jutemar <erik.jutemar@pagero.com>

#### Is this a new chart
No.

#### What this PR does / why we need it:
Allow agent pods to use the pod lifecycle postStart. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by arvindsv** [comment]
@varshavaradarajan knows this better.

/assign @varshavaradarajan

**Comment by varshavaradarajan** [comment]
@igno - thanks for the PR! The changes look good. Please update the changelog and we can merge this.

**Comment by igno** [comment]
@varshavaradarajan :+1:

File: stable/gocd/values.yaml [hunk 1]
```diff
@@ -246,6 +246,10 @@ agent:
   # preStop:
   #  - "/bin/bash"
   #  - "/disable_and_stop.sh"
+  # agent.postStart - array of commands to use in agent post-start lifecycle hook
```
Comment (arvindsv) [inline stable/gocd/values.yaml]: Is it really an array of commands? Or, one command with arguments? I feel that either the comment should change or the examples could be changed.

Comment (igno) [reply stable/gocd/values.yaml]: Indeed, it's a command with arguments. Updated the comments for both preStop and postStart.

**Pull request merged**